handover:
  ros__parameters:
    # Left arm poses (position in metres, orientation in rad from pendant)
    left_waypoint3_x:  -0.5285   #left post handover 
    left_waypoint3_y:  0.14535
    left_waypoint3_z:  0.10580
    left_waypoint3_rx: 3.799
    left_waypoint3_ry: -1.337
    left_waypoint3_rz: 1.881
    
    right_waypoint3_x:  -0.62240  #righthandover
    right_waypoint3_y:  0.03353
    right_waypoint3_z:  0.10272
    right_waypoint3_rx: 4.520
    right_waypoint3_ry: 1.006
    right_waypoint3_rz: -0.135

    right_waypoint4_x:  -0.62240 #right post handover 
    right_waypoint4_y:  -0.18753
    right_waypoint4_z:  0.10271
    right_waypoint4_rx: 4.520
    right_waypoint4_ry: 1.006
    right_waypoint4_rz: -0.136

    right_waypoint5_x:  -0.54174 #right lower object to final position
    right_waypoint5_y:  -0.18751
    right_waypoint5_z:  -0.08904
    right_waypoint5_rx: 3.135
    right_waypoint5_ry: 0.289
    right_waypoint5_rz: -0.094

    left_waypoint1 : [-0.2518866697894495, -1.2164602738669892, -2.014770746231079, -3.0570813618102015, -2.2569201628314417, 0.0768733024597168] # left starting position
    left_waypoint2 : [0.481362, -1.789312, -1.449321, -2.960602, -2.598097, 0.177675] # left handover position
    right_waypoint1 : [-3.2791, -1.6930, 2.1486, -2.3822, -2.7172, 1.8451] # right starting position
    right_waypoint2 : [-3.226939, -1.052608, 1.002168, -0.619068, -3.008948, 2.288825] # pre and post handover are the same, right prehandover
    right_lower_object_position : [-3.554538, -1.456826, 1.396787, -1.570447, -1.571320, 1.413891] # just before lowering

    # Tuning parameters
    approach_distance:  0.07
    right_y_offset:     0.15
    left_gripper_pin:   13
    right_gripper_pin:  12
    cartesian_waypoints: 20